packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Created by pyp2rpm-3.3.7
+%global pypi_name pulp-cli-deb
+
+Name:           %{?scl_prefix}python-%{pypi_name}
+Version:        0.0.1
```

Comment:
Since the `0.0.2` release is now available, (and it is the first release providing enough functionality for a basic sync workflow), we might consider skipping straight to this version.

This would require downloading the correct pypi package archive (https://pypi.org/project/pulp-cli-deb/) and re-adding it to git-annex.